api/references/when-clause-contexts.md

```diff
@@ -16,37 +16,130 @@ For example, VS Code uses when clauses to enable or disable command keybindings,
                    "when": "debuggersAvailable && !inDebugMode" },
 ```
 
-Above the built-in **Start Debugging** command has the keyboard shortcut `kb(workbench.action.debug.start)`, which is only enabled when there is an appropriate debugger available (context `debuggersAvailable` is true) and the editor isn't in debug mode (context `inDebugMode` is false).
+Above, the built-in **Start Debugging** command has the keyboard shortcut `kb(workbench.action.debug.start)`, which is only enabled when there is an appropriate debugger available (context key `debuggersAvailable` is true) and the editor isn't in debug mode (context key `inDebugMode` is false).
 
-## Conditional operators
+## Composing when-clauses with operators
 
-For conditional expressions, you can use the following conditional operators:
+A when-clause can consist of a single context key, e.g., `inDebugMode`, or can use various operators to express more nuanced editor state.
```

Comment:
Not sure I see the need to change "when clause" to "when-clause". Why do you think we need to add the hyphen?
Also we prefer to not use latin abbreviations, use "for example" instead

Reply:
Initially I thought using `"when" clause` might be a good idea since it matches what's in the JSON keybindings file but searching for `"when" clause`, I only found one instance in vscode-docs so I'm reluctant to go through all the docs for this.
I think leaving it as `when clause` is fine.

Reply:
I do see that we sometimes use "`when` clause", using backtick for an inline code block styling but that is also rare and when we are talking about the attribute in a JSON example